packages/example/src/components/modals/GenericModal.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/native';
+import React from 'react';
+import { View, ModalProps } from 'react-native';
+import { Typography } from '../../design-system/components/Typography';
+import { Spacer } from '../../design-system/components/Spacer';
+
+type CustomModalProps = ModalProps & {
+  isVisible: boolean;
+  children: React.ReactNode;
+  title: string;
+};
+
+export const GenericModal: React.FC<CustomModalProps> = ({ isVisible, children, title }) => {
```

Comment:
This is a very common misconception with typings haha

```suggestion
export const GenericModal = ({ isVisible, children, title }: CustomModalProps) => {
```

It's a strange practice that was propagated with the React Typescript template some time ago: https://github.com/facebook/create-react-app/pull/8177

Reply:
Oops, indeed, I correct it